TestFoundation/TestNSJSONSerialization.swift

```diff
@@ -956,11 +956,524 @@ extension TestNSJSONSerialization {
         }
     }
     
-    private func removeTestFile(_ location: String) {
+    fileprivate func removeTestFile(_ location: String) {
         do {
             try FileManager.default.removeItem(atPath: location)
         } catch _ {
             
         }
     }
 }
+
+//MARK: - JSONObjectWithStreams
```

Comment:
Modifying the existing test functions to include both JSONObject with data and with stream would look something like this:

` //MARK: - Object Deserialization
    func test_deserialize_emptyObject() {
        let subject = "{}"
        do {
            guard let data = subject.data(using: .utf8) else {
                XCTFail("Unable to convert string to data")
                return
            }
            // Test with Data
            let t = try JSONSerialization.jsonObject(with: data, options: [])
            let resultWithData = t as? [String: Any]
            //Test with Stream
            let inputStream: InputStream = InputStream(data: data)
            inputStream.open()
            let resultWithStream = try JSONSerialization.jsonObject(with: inputStream, options: []) as? [String: Any]
            inputStream.close()
            XCTAssertEqual(resultWithData?.count, 0)
            XCTAssertEqual(resultWithStream?.count, 0)
        } catch {
            XCTFail("Error thrown: \(error)")
        }
    }
`

Would something like this be ok?